docs/DeveloperGuide.md

```diff
@@ -60,6 +60,26 @@ The figure below represents the sequence diagram when the user wants to add a st
 
 <>
 
+### Model Component
+The `model` package contains the `InstrumentManager` class and the main parent class `Instrument`. It is defined
+in `InstrumentManager.java` and `Instrument.java` respectively. This figure below represents the class diagram of 
+how the different class work together:
+
+<>
+
+The `Model` component:
+
+1. Stores the instrument data through `Instrument` objects which are contained in the `InstrumentManager`
+2. Contains an abstract parent `Instrument` class. The 4 child sub-instrument classes `Crypto`, `Etf`, `Forex` and 
+`Stock` implements the Overridden methods (e.g. toList()).
+3. Contains the `InstrumentManager` which manages the list of instruments (e.g. add a new instrument to the list).
+4. Does not depend on any of the other components (as the `Model` component represents data on the instruments and can 
+exist on its own).
+
+The figure below represents the sequence diagram when the user wants to add a crypto:
```

Comment:
Can remove first, I notice the sequence diagrams for parser may also show an equivalent like add stock. But I think the sequence diagram for adding an instrument seems more appropriate here. What do you think? Do you have an alternative seq diagram we can show for parser?

Reply:
ur diagram would be between parser and command and mine would be passed from command right? Maybe can show a different functionality for this component like delete an instrument instead. But i think both would need a sequence diagram? Ill delete this part first then

Reply:
Yea I think the seq diagram maybe we can think of better functionality, add functionality is definitely the hardest, maybe both parser and instrument we can go for something simpler like delete, list or edit